apps/dot_ext/views/authorization.py

```diff
@@ -291,12 +295,42 @@ def dispatch(self, request, uuid, *args, **kwargs):
 class TokenView(DotTokenView):
     @method_decorator(sensitive_post_parameters("password"))
     def post(self, request, *args, **kwargs):
+        """Subclass of oauth2_provider.views.base.TokenView
+        to support checking application active/DAG status and
+        modification of the access token to include access
+        grant expiry.
+
+        RETURN:
+            response: HTTPResponse containing serialized access
+            token response.
+        """
         try:
-            validate_app_is_active(request)
+            dag = validate_app_is_active(request)[1]
+            if dag is not None and dag.expiration_date is not None:
+                dag_expiry = strftime('%Y-%m-%d %H:%M:%S', dag.expiration_date.timetuple())
+            else:
+                dag_expiry = None
         except (InvalidClientError, InvalidGrantError) as error:
             return json_response_from_oauth2_error(error)
 
-        return super().post(request, args, kwargs)
```

Comment:
What was `super().post` formerly doing and why don't we need it anymore? (Mostly for my own learning here)

Reply:
It's invoking the method from the parent class being subclassed. What I added is mostly taken from the parent class, so that the call to it is no longer necessary.